src/functionalTest/resources/features/F-702 - updateCasePaymentOrder/F-702.feature

```diff
@@ -0,0 +1,97 @@
+@F-702
+Feature: UPDATE Case Payment Order Endpoint
+
+  Background:
+    Given an appropriate test context as detailed in the test data source,
+    And a user with [an active profile in CCD],
+    And a successful call [to create a case payment order for the created case] as in [Prerequisite_Create_CPO],
+
+  @S-702.1
+  Scenario: AC1: Successfully allow the update of a previously created payment order in the Case Payment Order database
+    When a request is prepared with appropriate values
+    And the request [intends to update a previously created payment order]
+    And the request [contains all the mandatory parameters]
+    And it is submitted to call the [updateCasePaymentOrder] operation of [Case Payment Orders Microservice]
+    Then a positive response is received
+    And the response [contains a 200 success OK code]
```

Comment:
I think the Update response is now a 202 ?

Reply:
amended